src/components/Author.svelte

```diff
@@ -17,39 +18,37 @@
 	<div class="heading-wrapper">
 		<div class="author-heading">
 			<img
-				class="rounded-circle gray-scale"
+				class="gray-scale"
 				alt={author.name}
-				src={author.imageUrl
-					? urlFor(author.imageUrl).size(75, 75).quality(75).url()
-					: 'https://media.istockphoto.com/id/1214428300/vector/default-profile-picture-avatar-photo-placeholder-vector-illustration.jpg?s=612x612&w=0&k=20&c=vftMdLhldDx9houN4V-g3C9k0xl6YeBcoB_Rk6Trce0='}
-				height="75"
-				width="75"
+				src={author.profileImage}
+				height="25%"
+				width="25%"
 			/>
 			<div class="author-details">
 				<h1 class="mdc-typography--headline5" style="margin-bottom: 0;">{author.name}</h1>
 			</div>
 		</div>
-		<div>
+		<div class="author-details">
 			<p
 				class="mdc-typography--subtitle1"
 				style="margin-bottom: 0; text-align: left; font-style: italic; "
 			>
-				{author.title ? author.title : ''}
+				{author.title || ''}
```

Comment:
Strengt tatt en endring av logikk (`?` sjekker nullish, `||` falsy), men jeg kommer ikke på noen tilfeller hvor det skulle bli feil. :)

Reply:
Ikke jeg heller, men kan la den være for å ikke uheldigvis skulle endre på noe 😄